### PR TITLE
Modify wording, run-on sentence, contractions

### PR DESCRIPTION
**Pull request by zachmmeyer**

### CODE REVIEW

**Review by twalton83 — CHANGES_REQUESTED** [review]
Looks good except for one small thing!

File: javascript/introduction/introduction.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 
 The last thing you'll do is a final project which integrates everything you've learned in all the courses of this curriculum. This is the kind of project that you'll be telling employers all about and a great chance to prove that you're now, officially, a serious web developer.  
 
 ### Format
 
-There's a lot to cover, but it's been broken up into bite-sized lessons and their accompanying projects. The projects will give you a chance to apply what you've learned and to show what you're capable of. After a few of them, you'll really start getting the hang of things.  
+There is a lot to cover, but this course been broken up into bite-sized lessons and their accompanying projects. These projects will give you a chance to apply what you have learned and to show what you are capable of. After a few of them, you'll really start getting the hang of things.
```
[the remark this answers is not in the document]
Comment (zachmmeyer) [reply javascript/introduction/introduction.md:14]: There has been a lot to cover? Nothing of the course has been covered yet. Unless I'm misunderstanding. There will be a lot to cover might be more appropriate.

[the remark this answers is not in the document]
Comment (twalton83) [reply javascript/introduction/introduction.md:14]: Sorry, this was unclear. "this course been broken up" -> "has been broken up"

[the remark this answers is not in the document]
Comment (zachmmeyer) [reply javascript/introduction/introduction.md]: Thank you!

**Review by zachmmeyer — COMMENTED** [review]
Made requested changes

File: javascript/introduction/introduction.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 
 The last thing you'll do is a final project which integrates everything you've learned in all the courses of this curriculum. This is the kind of project that you'll be telling employers all about and a great chance to prove that you're now, officially, a serious web developer.  
 
 ### Format
 
-There's a lot to cover, but it's been broken up into bite-sized lessons and their accompanying projects. The projects will give you a chance to apply what you've learned and to show what you're capable of. After a few of them, you'll really start getting the hang of things.  
+There is a lot to cover, but this course has been course been broken up into bite-sized lessons and their accompanying projects. These projects will give you a chance to apply what you have learned and to show what you are capable of. After a few of them, you'll really start getting the hang of things.
```
Comment (twalton83) [inline javascript/introduction/introduction.md:14]: So close.. "this course has been course been"

Comment (zachmmeyer) [reply javascript/introduction/introduction.md:14]: Man I am having trouble with git hub. All the colors.

**Review by twalton83 — APPROVED** [review]
This was a journey.

**Pull request merged**